### PR TITLE
update docs and screenshots around iOS notifications / certificates

### PR DESCRIPTION
**Pull request by chrisfromredfin**
#### Summary
As the UI of Apple Developer has changed, the screenshots and process for generating a push notifications certificate have also changed. This PR updates both the screenshots and process for generating a certificate as well as using Keychain Access and openssl from the CLI to generate a certificate that works with mattermost-push-proxy.

### CODE REVIEW

File: site/content/contribute/mobile/push-notifications/service.md [hunk 1]
```diff
@@ -72,7 +72,9 @@ and running the service under a `mattermost-push-proxy` user account with limite
 
 ### Set up Mattermost Push Notification Service to Send iOS Push Notifications
 
-- Double click the **Distribution Certificate** generated while [Setup Push Notifications for iOS](/contribute/mobile/push-notifications/ios) to add it to your Keychain Access. Go to **Keychain Access**, select the **login** keychain and **My Certificates** from the side menu.
+- Double click the **Push Notifications Certificate** generated and downloading while [Setting up Push Notifications for iOS](/contribute/mobile/push-notifications/ios) to add it to your Keychain Access (downloads by default as aps.cer).
```
Comment (enahum) [inline site/content/contribute/mobile/push-notifications/service.md:75]:
```suggestion
- Double click the **Push Notifications Certificate** generated and downloaded while [Setting up Push Notifications for iOS](/contribute/mobile/push-notifications/ios) to add it to your Keychain Access (downloads by default as aps.cer).
```

**Review by enahum — CHANGES_REQUESTED** [review]
one small typo, the rest looks good.

Thanks @chrisfromredfin

**Review by enahum — APPROVED** [review]

**Comment by jfrerich** [comment]
@justinegeffen, gentle ping for review!

File: site/content/contribute/mobile/push-notifications/service.md [hunk 2]
```diff
@@ -182,6 +191,13 @@ panic: Failed to load the apple pem cert err=failed to parse PKCS1 private key f
 6. Verifying the certificate works with apple:
   - `openssl s_client -connect gateway.push.apple.com:2195 -cert aps_production.pem -key aps_production_priv.pem`
 
-### Reporting issues 
+##### DeviceTokenNotForTopic
+
+**For iOS / Apple Push Notifications**: If the logs are reflecting DeviceTokenNotForTopic (error 400)
+this may be because you're using an older / previous Device ID. Re-run the queries you need to get device IDs and test.
+
+This could also be because you generated a certificate for the wrong bundle ID. The bundle ID used in mattermost-push-proxy.json should be the same one as the app and the same that you generated the certificate for.
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply site/content/contribute/mobile/push-notifications/service.md:187]: @jfrerich, I don't know if my rewording makes sense - LMK what the paragraph refers to. :)

**Review by justinegeffen — COMMENTED** [review]
I've added some questions around the wording suggestions. Once you've had a look please feel free to request another review, or alternatively just let me know and I can create a new PR for additional changes.

**Comment by chrisfromredfin** [comment]
!Ping another reviewer?

**Comment by justinegeffen** [comment]
Hi @chrisfromredfin! I've gone ahead and committed my suggestions. If you're open to reviewing the doc and letting me know if it all makes sense then we can go ahead and merge. :)

File: site/content/contribute/mobile/push-notifications/service.md [hunk 3]
```diff
@@ -182,6 +191,13 @@ panic: Failed to load the apple pem cert err=failed to parse PKCS1 private key f
 6. Verifying the certificate works with apple:
   - `openssl s_client -connect gateway.push.apple.com:2195 -cert aps_production.pem -key aps_production_priv.pem`
 
-### Reporting issues 
+##### DeviceTokenNotForTopic
+
+**For iOS / Apple Push Notifications**: If the logs are reflecting DeviceTokenNotForTopic (error 400)
```
[the remark this answers is not in the document]
Comment (chrisfromredfin) [reply site/content/contribute/mobile/push-notifications/service.md]: 👍🏼

**Comment by chrisfromredfin** [comment]
I was just wondering about this PR today! What crazy timing. All of this looks fantastic to me!

**Comment by jasonblais** [comment]
@justinegeffen is this one then ready to merge?

@chrisfromredfin thanks for your contribution! :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**